alerting/src/main/kotlin/org/opensearch/alerting/workflow/CompositeWorkflowRunner.kt

```diff
@@ -226,11 +227,6 @@ object CompositeWorkflowRunner : WorkflowRunner() {
             )
         }
         workflowRunResult.executionEndTime = Instant.now()
-
-        val sr = SearchRequest(dataSources!!.alertsIndex)
-        sr.source().query(QueryBuilders.matchAllQuery()).size(10)
-        val searchResponse: SearchResponse = monitorCtx.client!!.suspendUntil { monitorCtx.client!!.search(sr, it) }
-        searchResponse.hits
```

Comment:
Was this dev code that was accidentally left over?

Reply:
yes